chanzuckerberg/single-cell-data-portal
# feat: Add curator name to a collection

### Reviewers
**Functional:** @seve

**Readability:** @ebezzi 

---

## Changes
- add database migration to add curator to collection info
- add curator name to the response of GET Collection
- add curator name from the id token to the PUT request when modifying a collection.

## Definition of Done (from ticket)
- The curator name is returned

## QA steps (optional)
https://add-curator-name-frontend.rdev.single-cell.czi.technology/
1. using the portal website create a collection. You can see `curator_name` is added to the payload when creating the collection
1. Navigate to the collection page. You will see `curator_name` in the GET response for the collection
1. Now modify the collection details. You can see `curator_name` is added to the payload when creating the collection.
1. Publish the collection, and then navigate to the collection page. You will see `curator_name` in the GET response for the collection.
